src/test/java/com/synacy/lesson01/exercises/ComplexNumberTest.java

```diff
@@ -0,0 +1,42 @@
+package com.synacy.lesson01.exercises;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class ComplexNumberTest {
+
+    @Test
+    public void add_shouldAddTwoComplexNumbersAndReturnAComplexNumber() throws Exception {
```

Comment:
A better naming without the use of conjunction ***and*** would be"add_shouldReturnTheSumOfTwoComplexNumbers()" :smile: